common/static/common/js/spec/components/list_spec.js

```diff
@@ -0,0 +1,67 @@
+define(['jquery', 'backbone', 'underscore', 'common/js/components/views/list'],
+    function ($, Backbone, _, ListView) {
+        'use strict';
+        describe('ListView', function () {
+            var Model = Backbone.Model.extend({
```

Comment:
Never mind. I think it is okay if we agree to disagree and that you can leave this as is.